frontend/src/__tests__/components/table/TableQuickInput.test.js

```diff
@@ -1,56 +1,52 @@
 import React from 'react';
-import { mount } from 'enzyme';
-// import { ShortcutProvider } from '../../../components/keyshortcuts/ShortcutProvider';
-import { initialState as appHandlerState } from '../../../reducers/appHandler';
-import { initialState as windowHandlerState } from '../../../reducers/windowHandler';
-import { Provider } from 'react-redux';
-import configureStore from 'redux-mock-store';
-import merge from 'merge';
-import tableQuickInputProps from '../../../../test_setup/fixtures/table/table_quick_input.json';
+import { shallow, render } from 'enzyme';
+import renderer from 'react-test-renderer';
+import nock from 'nock';
+
+import quickInputData from '../../../../test_setup/fixtures/table/table_quickinput.json';
 import TableQuickInput from '../../../components/table/TableQuickInput';
-import { getSizeClass } from '../../../utils/tableHelpers'; // imported as it is passed as a prop..
-
-const mockStore = configureStore([]);
-tableQuickInputProps.getSizeClass = getSizeClass;
-const createStore = function(state = {}) {
-  const res = merge.recursive(
-    true,
-    {
-      appHandler: {
-        ...appHandlerState,
-        me: { timeZone: 'America/Los_Angeles' },
-      },
-      windowHandler: { ...windowHandlerState },
-    },
-    state
-  );
-
-  return res;
+
+const initialProps = {
+  ...quickInputData.basicProps,
+  addNotification: jest.fn(),
 };
-const initialState = createStore({
-  windowHandler: {
-    allowShortcut: true,
-    modal: {
-      visible: false,
-    },
-  },
-});
-const store = mockStore(initialState);
 
-tableQuickInputProps.selected = [];
+// leaving this so that I won't have to look it up again in case we need it
+// nock.recorder.rec({
+//   output_objects: true,
+// }) 
 
 describe('TableQuickInput', () => {
-  it('renders without errors with the given props', () => {
-    const wrapperTableCMenu = mount(
-      <Provider store={store}>
-        <TableQuickInput {...tableQuickInputProps} />
-      </Provider>
-    );
-    const html = wrapperTableCMenu.html();
+  beforeEach(() => {
+    const { docType, docId, tabId } = initialProps;
+
+    nock(config.API_URL)
+      .defaultReplyHeaders({ 'access-control-allow-origin': '*' })
+      .post(`/window/${docType}/${docId}/${tabId}/quickInput`)
+      .reply(200, quickInputData.data1);
 
-    expect(html).toContain(
-      `<form class="row quick-input-container"><div class="col-sm-12 col-md-3 col-lg-2 hint">(Press 'Enter' to add)</div><button type="submit" class="hidden-up"></button></form>`
+    nock(config.API_URL)
+      .defaultReplyHeaders({ 'access-control-allow-origin': '*' })
+      .get(`/window/${docType}/${docId}/${tabId}/quickInput/layout`)
+      .reply(200, quickInputData.layout1); 
+  });
+
+  it('renders without errors', () => {
+    const wrapperTableCMenu = shallow(
```

Comment:
const wrapperTableCMenu = shallow(<TableQuickInput {...initialProps} />);  // prettify it